geode-core/src/test/java/org/apache/geode/distributed/internal/membership/MembershipDependenciesJUnitTest.java

```diff
@@ -93,6 +93,7 @@
               .or(resideInAPackage("org.apache.geode.logging.internal.executors.."))
               .or(resideInAPackage("org.apache.geode.distributed.internal.tcpserver.."))
               .or(type(LocalHostUtil.class))
+              .or(type(JavaWorkarounds.class))
```

Comment:
Am I reading that this class is now an allowed dependency, because of its movement from geode-core into geode-common?

Reply:
Yes. Although this test should probably be written to specify what packages are allowed, not individual classes?

Reply:
@rhoughton-pivot Nope, just breaking the dependency the soon to be module `membership` would have on this class being in `geode-core`

@upthewaterspout I'll adjust the ArchUnit exception to be package based, which might necessitate an ArchUnit test in `geode-common` to cross check , but maybe that's another Story